packages/pages/src/vite-plugin/build/buildStart/rendering/templateUtils.test.tsx

```diff
@@ -16,7 +16,7 @@ const baseTemplateModule: TemplateModuleInternal<any, any> = {
   templateName: "template",
   config: {
     name: "name",
-    hydrate: true,
+    hydrate: false,
```

Comment:
Why did this need to change?

Reply:
If hydrate is true in the tests, it attempts to find the bundled files in dist to use, but since we don't have actual files with actual filePaths being tested, there are no bundled client files